README.md

```diff
@@ -44,7 +44,7 @@ brew install curl autoconf automake libtool pkg-config
 git clone https://github.com/openvenues/libpostal
 cd libpostal
 ./bootstrap.sh
-./configure --datadir=[...some dir with a few GB of space...]
+./configure --datadir=~/libpostaldata  [...some dir with a few GB of space or leave blank for default location]
```

Comment:
Personally I'm a bit more confused by that formulation. In Unix command-line usage, it's common to use square brackets to denote optional arguments or varargs. That would read to me as if "~/libpostaldata" is required plus some additional optional arguments, which is not the case.

There may be a better notation for this, but `--datadir` is essentially something I do want the user to think about, since the model files occupy 1.8G of space at present (maybe more in the future), and most developers not working in ML are unaccustomed to that large a requirement from an open-source library.

Especially when using AWS, the default datadir location (/usr/local/share) on most systems, might take up valuable space on the root volume when it most likely should be on a larger EBS data volume.

There wasn't really a sensible default that I could think of for this case (depends on the user's mounted volumes, directory structure, etc.) so we instead tell the user how much disk space is required and defer to their judgment.

In a future version we may use some type of runtime config instead so it's easier to build binaries, at which point this option could be removed altogether and we'd just have the user run a setup script to download the models instead of `make`, but there's still likely some non-automatic setup.

Reply:
I've been buried. I'll update the PR with documentation on passing in options as well.